src/pages/contact-us.tsx

```diff
@@ -0,0 +1,254 @@
+import React from "react"
+import { PageProps } from "gatsby"
+import {
+  Button,
+  CircularProgress,
+  FormControl,
+  FormControlLabel,
+  FormLabel,
+  MenuItem,
+  Radio,
+  RadioGroup,
+  TextField,
+  Typography,
+} from "@mui/material"
+import Grid from "@mui/material/Unstable_Grid2/Grid2"
+
+// Utils
+import getPageTitle from "@utils/getPageTitle"
+
+// Components
+import { LinkButton } from "@components/Button"
+import { Section } from "@components/Layout"
+
+export const Head = getPageTitle("Contact Us")
+export type ContactUsState = {
+  name: string
+  lastName: string
+  email: string
+  phoneNumber: string
+  personType: string
+  subject: string
+  body: string
+}
+
+const encode = (data: Record<string, string>) => {
+  return Object.keys(data)
+    .map((key) => encodeURIComponent(key) + "=" + encodeURIComponent(data[key]))
+    .join("&")
+}
+
+export default function ContactUsPage({ data }: PageProps) {
+  const [state, setState] = React.useState<ContactUsState>({
+    name: "",
+    lastName: "",
+    email: "",
+    phoneNumber: "",
+    personType: "",
+    subject: "",
+    body: "",
+  })
+
+  const [submitContent, setSubmitContent] = React.useState(
+    <Button type="submit" color="tertiary" variant="contained" fullWidth>
+      <b>Submit</b>
+    </Button>
+  )
+
+  const handleChange = (event: React.FormEvent) => {
+    const target = event.target as HTMLInputElement
+    setState({ ...state, [target.name]: target.value })
+  }
+
+  const handleSubmit = (event: React.FormEvent) => {
+    setSubmitContent(<CircularProgress color="primary" />)
+    fetch("/", {
+      method: "POST",
+      headers: { "Content-Type": "application/x-www-form-urlencoded" },
+      body: encode({ "form-name": "contact", ...state }),
```

Comment:
I'm curious how each of these fields are going to be parsed in the Netlify form but this should be fine for now!